### PR TITLE
Flutter Festival Türkiye

### LINKED ISSUE #109: Big Update for Flutter Festival Türkiye
[ ] New Firebase will be connected.
[ ] Date will be updated.

### PR DESCRIPTION
**Pull request by bgoktugozdemir**
* README update
* Firebase Update
* UI Fixes
* Dynamic Main Buttons
* New sponsor categories
* Web app config class
* Language support for sessions.
* Cleaning unnecessary dependencies and files

Fixes #109

### CODE REVIEW

File: lib/core/utils/image_assets.dart [hunk 1]
```diff
@@ -9,4 +9,7 @@ class ImageAssets {
       'images/activity_flow/line_indicator_bottom.png';
   static const String lineIndicatorTop =
       'images/activity_flow/line_indicator_top.png';
+
+  static const String languageEN = 'images/languages/en.png';
+  static const String languageTR = 'images/languages/tr.png';
```
Comment (mukireus) [inline lib/core/utils/image_assets.dart:14]: Language yerine flag tercih edebiliriz. 

```suggestion
  static const String flagGB = 'images/flags/gb.png';
  static const String flagTR = 'images/flags/tr.png';
```

File: lib/event_flow/ui/event_flow_section.dart [hunk 2]
```diff
@@ -249,8 +279,23 @@ class _EventFlowSessionPoint extends StatelessWidget {
         width: radius,
         child: DecoratedBox(
           decoration: BoxDecoration(shape: BoxShape.circle, color: pointColor),
+          child: Padding(
+            padding: const EdgeInsets.all(2),
```
Comment (mukireus) [inline lib/event_flow/ui/event_flow_section.dart:283]: Values ​​must be multiples of 4.
```suggestion
            padding: const EdgeInsets.all(4),
```

Comment (bgoktugozdemir) [reply lib/event_flow/ui/event_flow_section.dart:283]: I agree with you about it. But when I make It 4, it looks too small the understand the flag.

### 2px Padding

Screen Shot 2022-03-11 at 19 38 44
Screen Shot 2022-03-11 at 19 39 32


-----

### 4px Padding

|Screen Shot 2022-03-11 at 19 36 10
Screen Shot 2022-03-11 at 19 35 52

File: lib/core/utils/image_assets.dart (hunk 1 above)
Comment (bgoktugozdemir) [reply lib/core/utils/image_assets.dart]: I updated 🙌

**Review by mukireus — APPROVED** [review]

**Pull request merged**